Resources/Prototypes/GameRules/roundstart.yml

```diff
@@ -254,12 +254,73 @@
   parent: BaseGameRule
   components:
   - type: BasicStationEventScheduler
+    scheduledGameRules:
+    - id: ClosetSkeleton
+    - id: MouseMigration
+    - id: SlimesSpawn
+    - id: BluespaceArtifact
+    - id: ClericalError
+    - id: IonStorm
+    - id: KingRatMigration
+    - id: CockroachMigration
+    - id: SpiderClownSpawn
+    - id: VentClog
+    - id: GasLeak
+    - id: ImmovableRodSpawn
+    - id: BreakerFlip
+    - id: PowerGridCheck
+    - id: BluespaceLocker
+    - id: NinjaSpawn
+    - id: BureaucraticError
+    - id: DragonSpawn
+    - id: SpiderSpawn
+    - id: MimicVendorRule
+    - id: LoneOpsSpawn
+    - id: RevenantSpawn
+#    - id: MassHallucination    # What the fuck is this? Is it its own whoosiwhatsit and not an event?
```

Comment:
its a station event and idk why you commented it out

Reply:
Oh I see what happened. I skill issued and tried to add the tracking component and not the rule component. Easy fix.